main/python-bcrypt/template.py

```diff
@@ -0,0 +1,31 @@
+pkgname = "python-bcrypt"
+pkgver = "4.2.0"
+pkgrel = 0
+build_style = "python_pep517"
+hostmakedepends = [
+    "cargo",
+    "python-build",
+    "python-installer",
+    "python-setuptools-rust",
+]
+makedepends = ["python-devel", "rust-std"]
+checkdepends = ["python-pytest"]
```

Comment:
```suggestion
checkdepends = ["python-pytest"]
depends = ["python"]
```

needed for leaf python package